admin/jest.config.ts

```diff
@@ -0,0 +1,12 @@
+import type { Config } from 'jest';
```

Comment:
**note:** Create a separate Jest config for the frontend project to avoid sharing the same configuration between frontend and backend (as they might be very different).